django_addanother/widgets.py

```diff
@@ -1,15 +1,17 @@
 import django
 from django import forms
-
 from django.contrib.admin.views.main import IS_POPUP_VAR
 from django.template.loader import render_to_string
 from django.utils.safestring import mark_safe
 
 
 if django.VERSION < (1, 9):
     DEFAULT_ADD_ICON = 'admin/img/icon_addlink.gif'
+    DEFAULT_EDIT_ICON = 'admin/img/icon_changelink.gif'
 else:
     DEFAULT_ADD_ICON = 'admin/img/icon-addlink.svg'
+    DEFAULT_EDIT_ICON = 'admin/img/icon-changelink.svg'
+DEFAULT_EDIT_ICON_DISABLED = 'django_addanother/img/icon-changelink-disabled.svg'
```

Comment:
Why do we have an extra icon for disabled? Why not simply make it say 80% opacity?